contrib/build-wine/build-electrum-git.sh

```diff
@@ -19,28 +19,25 @@ set -e
 cd tmp
 
 
-for repo in Electron-Cash; do
-    if [ -d $repo ]; then
-	cd $repo
-	git pull
-	git checkout 3.3.1
-	cd ..
-    else
-	URL=https://github.com/Electron-Cash/$repo
-	git clone -b master $URL electrum # rest of script assumes the dir is called 'electrum'
-    fi
-done
-
+if [ -d electrum ]; then
```

Comment:
You should be renaming all your old dirs to Electron-Cash!

Why support this old name style?

Is it because other repos still use it?

Ah! Eureka!  That's why, huh?

Perhaps some day we can all agree to use Electron-Cash as a name...